src/http-gateways/path-gateway.md

```diff
@@ -205,6 +205,30 @@ This is a URL-friendly alternative to sending an [`Accept`](#accept-request-head
 - `format=json` → `Accept: application/json`
 - `format=cbor` → `Accept: application/cbor`
 
+## Query Parameters for CAR Requests
+
+The following query parameters are only available for requests made with either a `format=car` query parameter or an `Accept: application/vnd.ipld.car` request header. These parameters modify shape of the IPLD graph returned within the car file
+
+### `car-scope` (request query parameter)
+
+Optional, `car-scope=(block|file|all)` with default value 'all', describes the shape of the dag fetched the terminus of the specified path whose blocks are included in the returned CAR file after the blocks required to traverse path segments.
+
+`block` - Only the root block at the end of the path is returned After blocks required to verify the specified path segments.
+
+`file` - For queries that traverse UnixFS data, `file` roughly means return blocks needed to verify the end of the path as a filesystem entity. In other words, all the blocks needed 'cat' a UnixFS file at the end of the specified path, or to 'ls' a UnixFS directory at the end of the specified path. For all queries that do not reference non-UnixFS data, `file` is equivalent to `block`
```

Comment:
```suggestion
`file` - For `{path}` segments traversed which are enumerable, only the minimum blocks needed for traversal should be returned. The final referenced object should be returned fully, but links should not be followed. In the specific instance of UnixFS data, these semantics are equivalent to the use of an [InterpretAs](https://ipld.io/specs/selectors/) [unixfs](https://github.com/ipfs/go-unixfsnode/blob/f4d6da0dbdf3614269b3580eeebe2bb5245f09bc/signaling.go#L47-L48) for the path, and as "unixfs-preload" for the terminal object.
```

Reply:
@willscott I am afraid linking/referring to implementation details from  `go-unixfsnode` source code is not something we can do in specs. 

If there is no easier way, we need to define these selectors in the spec before refering to them.